deployment/armada/values.yaml

```
@@ -9,9 +9,12 @@ resources:
     memory: 512Mi
     cpu: 200m
 additionalLabels: {}
-terminationGracePeriodSeconds: 0
-replicas: 1
-
+terminationGracePeriodSeconds: 5
+replicas: 3
```

```suggestion
replicas: 1
```
Lets keep this 1 for quick start.